Update base-zh-TW.yaml

C0D1R opened the pull request:
Translate the entire English content of the mods section into Traditional Chinese.

C0D1R committed:
Update base-zh-TW.yaml

Translate the entire English content of the mods section into Traditional Chinese.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/tobspr-games/shapez.io?pullRequest=1615) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/tobspr-games/shapez.io?pullRequest=1615) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/tobspr-games/shapez.io?pullRequest=1615) it.</sub>

C0D1R committed:
Update base-zh-TW.yaml

Translate the entire English content of the backendErrors section into Traditional Chinese.
